src/DHTDevice.h

```diff
@@ -31,8 +35,24 @@ class DHTDevice: public Device {
 
         String status();
 
+        void collect(String *metrics);
```

Comment:
really needed in interface?

Reply:
yes, its removal in the .h file causes the following err in the .cpp file: `inherited member is not allowedC/C++(298)`